test/TestConfig/scripts/tools/getDependencies.pl

```diff
@@ -136,19 +136,21 @@
 			}
 		}
 
-		# TODO check samtools.jar file
+		# TODO check asmtools.jar file
 		if (index($jars_info[$i]{fname}, "asmtools") != -1) {
-			my $untar = `tar -zxvf $path/asmtools-6.0.tar.gz -C $path`;
-			if ($? != 0 ) {
-				print $untar;
-				die "ERROR: untar $path/asmtools-6.0.tar.gz failed"
+			if (! -e "$path/asmtools.jar"){
+				my $untar = `tar -zxvf $path/asmtools-6.0.tar.gz -C $path`;
```

Comment:
Variable name and command does not match.

Reply:
all the uncompress process is for get the final asmtools.jar. The archived file is `asmtools-6.0.tar.gz` but the file we want is `asmtools.jar`